test/test_ops.py

```diff
@@ -231,69 +231,63 @@ def test_variant_consistency_jit(self, device, dtype, op):
         for sample in samples:
 
             # Acquires variants to test
+            func = op.get_op()
             method = op.get_method()
             inplace = op.get_inplace()
-            variants = (v for v in (method, inplace) if v is not None)
-
-            # Adds function variant to variant list
-            # TODO: inplace tests currently fail
-            # variants = (v for v in (op, method, inplace) if v is not None)
-            variants = (v for v in (op, method) if v is not None)
+            variants = {
+                'function': func, 'method': method,
+                # TODO: inplace tests currently fail
+                # 'inplace': inplace,
+            }
 
             # Test traced and scripted consistency
-            for variant in variants:
+            for func_type, variant in variants.items():
+                if variant is None:
+                    continue
+
                 # Create accessor for script function variant
-                if variant is op:
-                    name = op.name
-                    func_type = 'function'
-                elif variant is method:
-                    name = op.name
-                    func_type = 'method'
-                else:  # variant is inplace
-                    assert variant is inplace
-                    name = op.name + "_"
-                    func_type = 'inplace'
+                name = op.name + '_' if func_type == 'inplace' else op.name
```

Comment:
Not for this PR, but getting rid of these string manipulation hacks are the entire reason we're moving to OpInfos. We should probably let the name of the inplace variant of a function be requested from the OpInfo directly.